consensus/consensus-types/src/block.rs

```diff
@@ -125,6 +125,83 @@ impl Block {
         }
     }
 
+    /// Returns the number of proofs, the number of txns in the proofs, and the bytes of txns in the proofs
+    pub fn proof_statics(&self) -> (usize, usize, usize) {
```

Comment:
```suggestion
    pub fn proof_stats(&self) -> (usize, usize, usize) {
```
and everywhere else.